### PR TITLE
UID2-3407 Add details about logs in aws private operator

### PR DESCRIPTION
**Pull request by cYKatherine**

### CODE REVIEW

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:263]: We use present tense instead of future if possible. Suggest: 
When the operator instance has been deployed, the default log rotation settings are applied:
(even though this has passive in the first part, I think it's the clearest.)

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:264]: Can we say this some other way? But I don't totally understand it. And, ending punctuation, and present tense. Also ideal not to start a sentence with a number. A couple of guesses:
- Log rotations are archived for 30 instances.
- Back log rotations are stored for 30 instances before deletion.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:265]: Suggest:
- If there are no abnormal log entries, logs are rotated daily, so log history is equivalent to 30 days of data.

Also this made me wonder what would happen if it's abnormal. That's answered in the next point but, actually, it's just if it's abnormally large. So perhaps we could say that:
- If log entries are not abnormally large, (etc)

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:266]: (also space between number and MB) Suggest:

- If log entries are very large, and the log size reaches 30 MB within a 24-hour period, the log is rotated at that point.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:267]: Suggest:

- The following are the default logrotate settings, defined in `/etc/logrotate.d/uid2operator.conf`:

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:282]: Linux, initial cap.
Suggest:
For a detailed explaination of this config, see [logrotate(8) - Linux man page](https://linux.die.net/man/8/logrotate) or run `logrotate` man in the Linux environment.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:285]: Suggest:
- The logrotate generates the following script in /etc/cron.daily by default:

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:296]: - We put the following script in /etc/cron.d so that the logrotate check is run every minute:

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:304]: Suggested mods for this line and the two below:

- This is the default setting for these reasons:
   - So that the maxsize condition is checked frequently
    - The command refers to /var/lib/logrotate/logrotate.status to check the log status and see if it has reached the daily rotation time, so that it won't make an extra rotation.

Note I'm not sure on the above... I don't entirely understand it, particularly the last bit "so it won't make an extra rotation"... just looking at the language, not the technicalities. So, your call.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:308]: ### How to change the log rotation schedule

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:309]: To change the log rotation schedule, update the `etc/logrotate.d/uid2operator.conf` file.

Follow the instructions in the logrotate documentation: see [logrotate(8) - Linux man page](https://linux.die.net/man/8/logrotate).

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:311]: :::note
The service does NOT need to be restarted to pick up the change.
:::

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:314]: - To see a detailed explanation of what will be rotated, run `sudo logrotate -f /etc/logrotate.conf --debug`.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:315]: - To run one iteration of logrotate manually without changing the scheduled interval, run `sudo logrotate -f /etc/logrotate.conf --force`.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:316]: - To reload syslog-ng, run `sudo /usr/sbin/syslog-ng-ctl reload`.
(this is good, just add ending punctuation)

File: docs/guides/operator-guide-aws-marketplace.md
Comment (genwhittTTD) [inline docs/guides/operator-guide-aws-marketplace.md:320]: Curious where this email address came from. Just checking. Also, let's make it into a link so that the TTD email address isn't so obvious. Suggest:

If you have trouble subscribing to the product or deploying it, [contact us](mailto:aws-mktpl-uid@thetradedesk.com).

**Review by genwhittTTD — CHANGES_REQUESTED** [review]
Quite a few wording mods. LMK if any questions. Happy to discuss and should definitely have another review.

File: docs/guides/operator-guide-aws-marketplace.md
Comment (cYKatherine) [reply docs/guides/operator-guide-aws-marketplace.md:320]: Checked with the team, Its a mailing list that goes to some of us who have related knowledge.

**Review by genwhittTTD — APPROVED** [review]

**Pull request merged**
